tools/lint/lint.py

```diff
@@ -146,6 +146,13 @@ def check_path_length(repo_root, path):
     return []
 
 
+def check_file_type(repo_root, path):
+    # type: (str, str) -> List[rules.Error]
+    if os.path.islink(path):
+        return [rules.FileType.error(path, (path, "link"))]
```

Comment:
Perhaps
```suggestion
        return [rules.FileType.error(path, (path, "symlink"))]
```
I haven't really used the terminology "link" and probably wouldn't be able to know right away what it is without context.

Reply:
Judging only by the name of the method, I assumed it would also identify hardlinks. [That seems *not* to be the case](https://docs.python.org/2/library/os.path.html#os.path.islink), so maybe "symlink" (or even "symbolic link") would be better.

Reply:
No, hardlink is more transparent to users and is just a normal file to most userland applications; you'd need to use special syscalls (inode stuff?) to tell.